core/server/worker/src/test/java/alluxio/worker/block/DefaultBlockWorkerTest.java

```diff
@@ -668,7 +669,7 @@ private void cacheBlock(boolean async) throws Exception {
     long sessionId = mRandom.nextLong();
     // check that we can read the block locally
     try (BlockReader reader = mBlockWorker.createBlockReader(
-            sessionId, blockId, 0, false, null)) {
+            sessionId, blockId, 0, false, options)) {
       ByteBuffer buf = reader.read(0, ufsBlockSize);
       // alert: LocalFileBlockReader uses a MappedByteBuffer, which does not
       // support the array operation. So we need to compare ByteBuffer manually
```

Comment:
Could we use an option that doesn't contain `ufsPath` and `blockInUfsTier` for this? I had intended to test that a `BlockReader` can be constructed without falling back to ufs here.

Reply:
Feel free open a PR to correct this. I didn't go through the actual logic of the test.